node/node.go

```diff
@@ -345,50 +337,29 @@ func makeNode(cfg *config.Config,
 		channels[ch.ID] = ch
 	}
 
-	peerUpdates := peerManager.Subscribe()
 	stateSyncReactor := statesync.NewReactor(
 		genDoc.ChainID,
 		genDoc.InitialHeight,
 		*cfg.StateSync,
-		ssLogger,
+		logger.With("module", "statesync"),
 		proxyApp.Snapshot(),
 		proxyApp.Query(),
 		channels[statesync.SnapshotChannel],
 		channels[statesync.ChunkChannel],
 		channels[statesync.LightBlockChannel],
 		channels[statesync.ParamsChannel],
-		peerUpdates,
+		peerManager.Subscribe(),
 		stateStore,
 		blockStore,
 		cfg.StateSync.TempDir,
 		nodeMetrics.statesync,
 	)
 
-	// Optionally, start the pex reactor
-	//
-	// TODO:
-	//
-	// We need to set Seeds and PersistentPeers on the switch,
-	// since it needs to be able to use these (and their DNS names)
-	// even if the PEX is off. We can include the DNS name in the NetAddress,
-	// but it would still be nice to have a clear list of the current "PersistentPeers"
-	// somewhere that we can return with net_info.
-	//
-	// If PEX is on, it should handle dialing the seeds. Otherwise the switch does it.
-	// Note we currently use the addrBook regardless at least for AddOurAddress
-
 	pexReactor, err := createPEXReactor(logger, peerManager, router)
```

Comment:
The pex reactor is something we can still turn off right? Do we check the config anywhere? Or is it just `OnStart()`

Reply:
That configuration value isn't checked anywhere in the current iteration of the code.

Reply:
You mean `Config.P2P.PexReactor` isn't used at all? Can we change that and backport it to v0.35? Pex should be configurable

Reply:
#7198